phpBB/includes/constants.php

```diff
@@ -190,6 +190,7 @@
 define('BBCODE_ID_EMAIL', 10);
 define('BBCODE_ID_FLASH', 11);
 define('BBCODE_ID_ATTACH', 12);
+define('BBCODE_ID_MENTION', 13);
```

Comment:
How is this going to work out with users having existing custum_bbcodes? Do they start with ID 13? If so, this might cause problems I assume with those. Maybe even a idea to define it completly as custom bbcode instead of hardcoded in core

Reply:
I suppose that existing custom BBCodes don't have the hardcoded constant values.
But probably this constant can be removed entirely because it does not seem that I need to use it anywhere with the new parser. It could be suitable for deprecated stuff that is not necessary with the newly added BBCode.

Reply:
The issue is that the first created custom_bbcode is ID 13: 
![image](https://user-images.githubusercontent.com/65767/41282240-e76a695e-6e33-11e8-90ea-a5795dd1f0d9.png)
No idea if that can cause issues, but I can see it might happen.